src/Components/App/App.tsx

```diff
@@ -1,11 +1,12 @@
 import './App.scss';
 import React, { Component } from 'react'
 import NewPostForm from '../NewPostForm/NewPostForm'
+import Loading from '../Loading/Loading'
 import AllPosts from '../AllPosts/AllPosts'
 import Searchbar from '../Searchbar/Searchbar'
 import Nav from '../Nav/Nav'
 import { IPost } from '../../types'
-// import { Link } from 'react-router-dom';
+import { Route, Switch, Link } from 'react-router-dom';
```

Comment:
We might be able to take Link out since we aren't using it in this file

Reply:
Done in fix!